jenkins/automation/s3server/s3_ceph_testcase.groovy

```diff
@@ -0,0 +1,93 @@
+pipeline {	 
+    agent {
+		node {
+            // Agent created with 4GB ram/16GB memory in EOS_SVC_RE1 account 
+			label "s3-compatibility-test-ceph-rhev-7.9"
+            // Use custom workspace for easy troublshooting
+            customWorkspace "/root/compatability-test/${INTEGRATION_TYPE}"
+		}
+	}
+
+	options {
+        timeout(time: 240, unit: 'MINUTES')
+        timestamps()
+        ansiColor('xterm') 
+        buildDiscarder(logRotator(daysToKeepStr: '5', numToKeepStr: '5')) 
+    }
+    parameters {  
+        string(name: 'RGW_PORT', defaultValue: '30080', description: 's3-test rgw port', trim: true)
+        string(name: 'RGW_SERVICE_IP', defaultValue: '', description: 's3-test rgw ip address', trim: true)
+	    string(name: 'S3_TEST_REPO', defaultValue: 'https://github.com/ceph/s3-tests', description: 's3-test ceph repo', trim: true)
+        // we are using specific revision of 'https://github.com/ceph/s3-tests' for our tests  - default
+        string(name: 'S3_TEST_REPO_REV', defaultValue: 'bacab3cadfdd7c4ce1f03793239f919c0a3c09e2', description: 's3-test repo revision', trim: true)
+        choice(name: 'INTEGRATION_TYPE', choices: [ "ceph"], description: 'S3 Integration Type')
+	}
+
+    environment {
+        // This config file used for ceph compatibility tests
+        S3_TEST_CONF_FILE = "${INTEGRATION_TYPE}_${BUILD_NUMBER}.conf"
+    }
+ 	
+	stages {
+        stage ('Update Test Config') { 
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {              
+                    sh label: 'run compatibility test', script: '''
+                        #set +x
+                        hostentrycheck=$(grep -r "s3.seagate.com" /etc/hosts|grep -e s3.seagate.com -e $RGW_SERVICE_IP|wc -l)
```

Comment:
We can always clean up old entry with `s3.seagate.com` and add a new entry

Reply:
Removed some lines related to logic